packages/blade/src/components/BaseMenu/BaseMenuItem.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from '~components/Box';
+import BaseBox from '~components/Box/BaseBox';
+import { Checkbox } from '~components/Checkbox';
+import { Text } from '~components/Typography';
+import { size } from '~tokens/global';
+import { getMediaQuery, makeBorderSize, makeSize } from '~utils';
+import { getFocusRingStyles } from '~utils/getFocusRingStyles';
+import { makeAccessible } from '~utils/makeAccessible';
+import type { BladeElementRef } from '~utils/types';
+
+const StyledMenuItemContainer = styled.button<{ color?: 'negative' }>((props) => {
+  return {
+    borderWidth: makeBorderSize(props.theme.border.width.none),
+    display: 'flex',
+    flexDirection: 'column',
+    textAlign: 'left',
+    backgroundColor: 'transparent',
+    padding: makeSize(props.theme.spacing[2]),
+    borderRadius: makeSize(props.theme.border.radius.medium),
+    marginTop: makeSize(props.theme.spacing[1]),
+    marginBottom: makeSize(props.theme.spacing[1]),
+    textDecoration: 'none',
+    cursor: 'pointer',
+    width: '100%',
+    [`@media ${getMediaQuery({ min: props.theme.breakpoints.m })}`]: {
+      padding: makeSize(props.theme.spacing[3]),
+    },
+    '&:focus-visible': getFocusRingStyles({ theme: props.theme }),
+    '&:hover:not([aria-disabled=true]), &.has-submenu-open': {
```

Comment:
Why are we checking for aria-disabled? Lets instead check for the disabled prop directly?

Reply:
Been preferring the CSS selectors over JS checks. One check won't make much difference but seems like a good practice.